src/twig/package.json

```diff
@@ -14,9 +14,14 @@
     "@nypl/design-system-icons": "file:../icons",
     "chai": "^4.2.0",
     "mocha": "^6.2.2",
-    "puppeteer": "^2.0.0"
+    "puppeteer": "^2.0.0",
+    "webpack-cli": "^3.3.11"
   },
   "publishConfig": {
     "access": "public"
+  },
+  "dependencies": {
+    "extract-text-webpack-plugin": "^3.0.2",
+    "sass-loader": "^8.0.2"
```

Comment:
This PR is for the react components package, so are these updates necessary? And if so, what are they doing? Or is this similar to the Lerna hoisting you mentioned in a different comment?